lib/tasks/javascript.rake

```diff
@@ -130,6 +130,135 @@ def absolute_sourcemap(dest)
   end
 end
 
+def generate_admin_sidebar_nav_map
+  vague_categories = { "root" => [] }
+
+  admin_routes =
+    Rails
+      .application
+      .routes
+      .routes
+      .map do |route|
+        next if route.verb != "GET"
+        path = route.path.spec.to_s.gsub("(.:format)", "")
+        next if !path.include?("admin")
+        next if path.include?("/:") || path.include?("admin-login")
+        path
+      end
+      .compact
+
+  engine_routes =
```

Comment:
Quite annoying that engine routes are not included in `Rails.application.routes.routes`!